app/views/session/new.erb

```diff
@@ -2,5 +2,5 @@
 <form action='/session/new' method ='post'>
   E-Mail: <input type='email' name='email'>
   Password: <input type='password' name='password'>
-  <input type='submit' value='Sign in'>
+  <input type='submit' id='sign_in' value='Sign in'>
```

Comment:
I would try and make the text on the button unique to the page, i.e. have it descriptive ('sign in', 'register', 'create listing' instead of just 'submit' everywhere). By doing this, it's a) semantic and b) we don't need to have the ID just for testing purposes.